L1Trigger/TrackerDTC/interface/Settings.h

```diff
@@ -119,9 +117,26 @@ namespace trackerDTC {
 
     double bField() const { return bField_; }
 
+    // Analzyer
+
+    bool useMCTruth() const { return useMCTruth_; }
+    edm::InputTag inputTagTTClusterAssMap() const { return inputTagTTClusterAssMap_; }
+    std::string producerLabel() const { return producerLabel_; }
```

Comment:
in general it's better to return complex objects by const ref, then the code that calls the accessor can decide if it wants to make a copy or keep a reference